packages/opentelemetry-types/src/trace/Sampler.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SpanContext } from './span_context';
+
+/** This interface represent a sampler. */
+export interface Sampler {
```

Comment:
Just curious, who is meant to consume these typings (outside of this repo)? Is the point of these in case src files are switched away from typescript files? Not specific to this PR specifically, but to the repo in general.

Reply:
You may want to read discussion [here](https://github.com/open-telemetry/opentelemetry-js/issues/3).